app/org.chessmd.chessmd.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=ChessMD
+Exec=/usr/bin/flatpak run org.chessmd.chessmd
```

Comment:
NEVER EVER should there be "flatpak" here. Not sure what documentation to read that told you to do that, but it is very wrong.